pkg/lib/localstore/ftsearch/ftsearch.go

```diff
@@ -174,8 +175,17 @@ func (f *ftSearch) getTerms(qry string) []string {
 	return terms
 }
 
-func (f *ftSearch) doSearch(queries []query.Query) (results []string, err error) {
-	searchRequest := bleve.NewSearchRequest(bleve.NewDisjunctionQuery(queries...))
+func (f *ftSearch) doSearch(spaceID string, queries []query.Query) (results []string, err error) {
+	var rootQuery query.Query
+	disjunctionQuery := bleve.NewDisjunctionQuery(queries...)
```

Comment:
AFAIU 179-188 can be simplified
```
rootQuery := bleve.NewDisjunctionQuery(queries...)
if spaceID != "" {
		spaceQuery := bleve.NewMatchQuery(spaceID)
		rootQuery = bleve.NewConjunctionQuery(disjunctionQuery, spaceQuery)
}
```

Reply:
Thank you! Simplified the code by getting rid of disjunctionQuery variable